### PR TITLE
[force_ssl_bugfix] Corrected the setting name for force_ssl

### PR DESCRIPTION
**Pull request by oskarpearson**
The environment variable wasn't being honored by rails, and was defaulting to false.

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = Settings.force_ssl
+  config.force_ssl = Settings.rails.force_ssl
```
Comment (oskarpearson) [inline config/environments/production.rb]: see https://github.com/ministryofjustice/defence-request-service/blob/master/config/settings.yml#L6-L8 - where the force_ssl item is within the 'rails:' section

File: .env.development [hunk 2]
```diff
@@ -1,11 +1,13 @@
 # App settings
-export DISABLE_DASHBOARD_REFRESH=true
+DISABLE_DASHBOARD_REFRESH=true
 
 # Database settings
 DEFENCE_SOLICITOR_DATABASE_HOST=localhost
 DEFENCE_SOLICITOR_DATABASE_PASSWORD=""
 DEFENCE_SOLICITOR_DATABASE_USERNAME="postgres"
 
+FORCE_SSL=false
```
Comment (oskarpearson) [inline .env.development]: Question for everyone - given that the force_ssl option is only set in production.rb, do you think this is overkill?

Comment (oskarpearson) [inline .env.development]: https://github.com/ministryofjustice/defence-request-service/blob/master/config/environments/production.rb#L46 but not in https://github.com/ministryofjustice/defence-request-service/blob/master/config/environments/development.rb

**Comment by oskarpearson** [comment]
Answering my own questions above, I'm not going to set the variable for other environments. It's unnecessarily and breaks the principle of least surprise ("I changed this variable, but it doesn't have the impact I expected")

**Comment by novotnyjakub** [comment]
Can we set it false for `development` and `test` environments? 

I'm suspecting that may be the reason the build failed as well.

**Comment by oskarpearson** [comment]
Chatted to Jakub - I've not added to the development/test environments because only production.rb uses the environment variable

**Pull request merged**